src/content/en/tools/chrome-devtools/debug/remote-debugging/local-server.markdown

```diff
@@ -1,147 +1,137 @@
 ---
 layout: shared/narrow
-title: "Remote Access to Your Local Site"
-description: "The simplest way to test on a real device is to run your site in a web server and point your device to the IP address."
+title: "Access Local Server from Android Device"
+description: "Host a site on a development machine web server, and then access
+the content from an Android device."
 authors:
+  - kaycebasques
   - megginkearney
 published_on: 2015-04-14
-updated_on: 2015-04-14
+updated_on: 2016-04-08
 order: 2
 translation_priority: 1
 key-takeaways:
   local-server:
-    - "Use port forwarding when your device can't access your development server's network."
-    - "To reach content on a customized domain, use port forwarding in combination with a proxy server."
+    - Port forwarding enables you to view content from your development 
+      machine's web server on your Android device.
+    - If your web server is using a custom domain, you can set up your 
+      Android device to access the content at that domain with custom domain
+      mapping. 
 ---
 
-<p class="intro">
-  The simplest way to test on a real device is to run your site in a web server and point your device to the IP address.
-</p>
+<p class="intro">Host a site on a development machine web server, and then 
+access the content from an Android device.</p>
+
+You're developing a site. You want to host the site on your development
```

Comment:
This reads weirdly. Proposal: "When developing a site locally, it runs on your own development machine. By default, typing 'localhost' on your Android device would try to resolve a local server on your phone instead of showing the site running on your development machine, making debugging it on your phone impossible. But DevTools can work around this limitation with the help of port forwarding. This guide helps you to set it all up."